bulbs/styles/static/styles/bulbs-styles/components/ads.scss

```diff
@@ -0,0 +1,8 @@
+.dfp-slot-inread {
+  color: #7f7f7f;
+  font-family: Helvetica Neue;
```

Comment:
Nit picky but you should include the whole family as a fallback here: 

`font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;`